vignettes/gevp.Rmd

```diff
@@ -19,13 +19,13 @@ library(knitr)
 library(hadron)
 ```
 
-# Data Type \texttt{cf} for Correlation Functions
+# Data Type `cf` for Correlation Functions
 
-\texttt{hadron} provides a data type or better class for correlation
-functions and correlation matrices, which is called \texttt{cf}. There
+`hadron` provides a data type or better class for correlation
```

Comment:
does this render properly in the end?

Reply:
Sorry, not here, but when used within roxygen-generated docs?

Reply:
It is a markdown document, so using backticks is more proper than using LaTeX code. So for instance this won't work when it is rendered as HTML.

Reply:
So far I have the impression that Roxygen properly does Markdown.